index.html

```diff
@@ -240,6 +250,60 @@ <h2 class="my-project-name">Multi-Post Stories</h2>
       </div>
     </div>
   </section>
+
+  <div class="contact-form">
+
+      <h3 class="contact-form-title">CONTACT US</h3>
+
+      <p class="contact-form-title">If you have an application you are interested in developing, a feature that you
+         need built or a project that needs coding. I’d love to help with it</p>
+
+    <form action="https://formspree.io/f/moqrjakr" method="post">
+      
+         <input type="text" name="user-first-name" placeholder="Text Input" maxlength="30" required>
+      
+         <input type="email" name="user-email" placeholder="jeremy@gmail.com" class="name-ph-change" 
+         pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
```

Comment:
- [ ] I see you are validating the emails with `pattern` and `type="email"`.Both ways of validating email addresses have their pros and cons it is up to you to decide which one to use but in this case, the requirements request to use the `type="email"`. You should not try to use them both at the same time as this might induce a clash in browsers that support both features. Using type=”email” has the advantage that it is semantically correct both using the pattern attribute has the advantage that there are several easy-to-use polyfills on the web which ensures support for a greater range of audience.

## Example :
![image](https://user-images.githubusercontent.com/28109626/149241982-28321c3c-83fc-4c73-9959-8b1c3198fe16.png)

So please not use both at the same time, and for this occasion make use of type="email" to follow project requirements.

Reply:
OK. I let only the project´s requirement validation method for the e-mail field.